x/gov/keeper/grpc_query.go

```diff
@@ -167,6 +167,7 @@ func (q Keeper) Params(c context.Context, req *v1.QueryParamsRequest) (*v1.Query
 
 	ctx := sdk.UnwrapSDKContext(c)
 	params := q.GetParams(ctx)
+	params.MinDeposit = q.GetMinDeposit(ctx)
```

Comment:
Since you added a specific query for the min deposit, it looks weird to feed this parameter that is marked as deprecated. I assume this is for retro compatibility with clients right ?

Reply:
yes indeed

Reply:
OK then let's add a comment regarding that.